src/common/com/intellij/plugins/haxe/haxelib/HaxelibUtil.java

```diff
@@ -134,6 +138,10 @@ public static VirtualFile getLibraryRoot(@NotNull Sdk sdk, @NotNull String libNa
       if (s.isEmpty()) continue;
       if (s.startsWith("-D")) continue;
       if (s.startsWith("-L")) continue;
+      if (s.matches("Error: .*")) {
```

Comment:
When haxelib actually issues an error, we now present it to the user instead of trying to use it as a library path.